graphql/schema.graphql

```diff
@@ -316,6 +317,12 @@ input ProjectInput {
   repo: String!
 }
 
+input CopyProjectInput {
+  projectIdToCopy: String
```

Comment:
Since we need this to be able to copy the project and it missing produces and error, this should probably be required.